owasp-dependency-checker-suppressions.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
```

Comment:
Could you please add the commands to utilize this plug-in in the `Makefile`?

Reply:
it will run on the make install step as its plugged into the verify step of maven. Travis will run it during 
`mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V`